package.json

```diff
@@ -79,6 +79,7 @@
     ]
   },
   "dependencies": {
-    "html": "0.0.10"
+    "html": "0.0.10",
+    "react-element-to-jsx-string": "^2.6.1"
```

Comment:
I wonder if we can leverage that library too for our [`inspect`](https://github.com/producthunt/chai-enzyme/blob/master/src/CheerioTestWrapper.js#L22) functions. (different PR)